http/transport.go

```diff
@@ -27,8 +27,11 @@ import (
 	"github.com/weaveworks/flux/platform/rpc"
 )
 
+const URL_PREFIX = "/api/flux"
+
 func NewRouter() *mux.Router {
 	r := mux.NewRouter()
+	r = r.PathPrefix(URL_PREFIX).Subrouter()
```

Comment:
With a subrouter, check that the router lookup route-by-name thing we do in the client still works.